src/app/components/client/toolbar/UserMenu.module.scss

```diff
@@ -0,0 +1,94 @@
+@import "../../../tokens";
+
+$button-size: 42px;
+
+.trigger {
+  background: white;
+  border-radius: 50%;
+  border-style: none;
+  cursor: pointer;
+  height: $button-size;
+  padding: 0;
+  width: $button-size;
+
+  @media screen and (max-width: $screen-md) {
+    margin-right: $spacing-sm;
+  }
+
+  &:focus,
+  &:hover {
+    outline: 2px solid $color-purple-70;
+  }
+}
+
+.menuList {
+  border-radius: $border-radius-md;
+  box-shadow: $box-shadow-md;
+  font: $text-body-md;
+  list-style-type: none;
+  min-width: $content-xs;
+  outline: 0;
+  padding: $spacing-sm 0;
+  position: relative;
+  right: 0;
+
+  &::before {
+    $tooltip-size: 12px;
+    background-color: white;
+    content: "";
+    height: $tooltip-size;
+    position: absolute;
+    right: calc($spacing-md + $button-size * 0.5);
+    top: 0;
+    transform: translateY(-50%) rotate(45deg);
+    width: $tooltip-size;
+
+    @media screen and (max-width: $screen-md) {
+      right: calc($button-size * 0.5);
+    }
+  }
+}
+
+.menuItemWrapper {
+  cursor: pointer;
+  padding: $spacing-sm calc($spacing-md * 1.5);
+  position: relative;
+
+  &:focus {
+    outline: none;
+    background: $color-grey-10;
+  }
+
+  &:first-of-type {
+    margin-bottom: $spacing-sm;
```

Comment:
Bringing up my usual comment when I see margins (down below as well): any reason it couldn't be done with a padding or gap on the parent element? Makes it more predictable which elements are responsible for the layout.

Reply:
Gotcha - I take it the `:after` can't have its own marging applied then? In any case, maybe you could add a comment explaining what the margin is there for.

Reply:
Unfortunately, adding a margin to the pseudoelement would result in a visually unpleasant state when hovering over the header item. Added a comment with [93ca023](https://github.com/mozilla/blurts-server/pull/3326/commits/93ca0232a3848dae2ab86352869c02175cd56806).